package-lock.json

```diff
@@ -1,8 +1,15395 @@
+            "integrity": "sha512-xkNcLAn/wZaX14RPlwizcKicDk9G3F8m2nU3L7Ukm5zBgTwiT0wsoFAHx9Jq56fJA1z/7uKGtCRu16sOUCLIHQ==",
+            "dev": true,
+            "license": "BSD-3-Clause",
+            "dependencies": {
+                "type-detect": "4.0.8"
+            }
+        },
+        "node_modules/@sinonjs/fake-timers": {
+            "version": "6.0.1",
+            "resolved": "https://registry.npmjs.org/@sinonjs/fake-timers/-/fake-timers-6.0.1.tgz",
+            "integrity": "sha512-MZPUxrmFubI36XS1DI3qmI0YdN1gks62JtFZvxR67ljjSNCeK6U08Zx4msEWOXuofgqUt6zPHSi1H9fbjR/NRA==",
+            "dev": true,
+            "license": "BSD-3-Clause",
+            "dependencies": {
+                "@sinonjs/commons": "^1.7.0"
+            }
+        },
+        "node_modules/@slack/events-api": {
+            "version": "2.3.2",
+            "resolved": "https://registry.npmjs.org/@slack/events-api/-/events-api-2.3.2.tgz",
+            "integrity": "sha512-jIKLsvPn4Ute/WUSlVwTW3o58S0PRtZN+QO3El0MN10IFzZ6iEGXvvhjXGLovUydEvucO/Lu5cISm5+tKGAiBw==",
+            "license": "MIT",
+            "dependencies": {
+                "@types/debug": "^4.1.4",
+                "@types/express": "^4.17.0",
+                "@types/lodash.isstring": "^4.0.6",
+                "@types/node": ">=4.2.0 < 13",
+                "@types/yargs": "^13.0.0",
+                "debug": "^2.6.1",
+                "lodash.isstring": "^4.0.1",
+                "raw-body": "^2.3.3",
+                "tsscmp": "^1.0.6",
+                "yargs": "^6.6.0"
+            },
+            "bin": {
+                "slack-verify": "dist/verify.js"
+            },
+            "engines": {
+                "node": ">=4.2.0"
+            },
+            "optionalDependencies": {
+                "express": "^4.0.0"
+            }
+        },
+        "node_modules/@slack/events-api/node_modules/@types/node": {
+            "version": "12.12.35",
+            "resolved": "https://registry.npmjs.org/@types/node/-/node-12.12.35.tgz",
+            "integrity": "sha512-ASYsaKecA7TUsDrqIGPNk3JeEox0z/0XR/WsJJ8BIX/9+SkMSImQXKWfU/yBrSyc7ZSE/NPqLu36Nur0miCFfQ==",
+            "license": "MIT"
+        },
+        "node_modules/@slack/logger": {
+            "version": "2.0.0",
+            "resolved": "https://registry.npmjs.org/@slack/logger/-/logger-2.0.0.tgz",
+            "integrity": "sha512-OkIJpiU2fz6HOJujhlhfIGrc8hB4ibqtf7nnbJQDerG0BqwZCfmgtK5sWzZ0TkXVRBKD5MpLrTmCYyMxoMCgPw==",
+            "license": "MIT",
+            "dependencies": {
+                "@types/node": ">=8.9.0"
+            },
+            "engines": {
+                "node": ">= 8.9.0",
+                "npm": ">= 5.5.1"
+            }
+        },
+        "node_modules/@slack/types": {
+            "version": "1.10.0",
+            "resolved": "https://nexus.global.picnicinternational.com/repository/npm/@slack/types/-/types-1.10.0.tgz",
```

Comment:
CI is failing because it's trying to fetch packages from the Nexus but it doesn't have access.

Reply:
Fixed in 396e8e3fdc9572226c633cb1790a9df4060be7b4